context.go

```diff
@@ -99,28 +100,40 @@ func (context *Context) start(ws *websocket.Conn) *Sender {
 }
 
 // Opens a BLIP connection to a host.
-func (context *Context) Dial(url string, origin string) (*Sender, error) {
-	config, err := websocket.NewConfig(url, origin)
-	if err != nil {
-		return nil, err
-	}
-	return context.DialConfig(config)
+func (context *Context) Dial(url string) (*Sender, error) {
+	return context.DialConfig(&DialOptions{
+		URL: url,
+	})
```

Comment:
Does the new library not have the concept of an origin for a websocket connection, or is it just the case that we're not using it in a meaningful way anyway?

Reply:
We just weren't using it previously (It's for a CORS check) - the library does have opt-in support for specifying Origin patterns if we were to require it in the future.